addons/sale/static/src/js/product/product.scss

```
@@ -32,5 +40,9 @@
 }
 
 .o_sale_product_configurator_price {
-    width: 160px;
+    @include media-breakpoint-up(md) {
+        width: 160px;
+    }
+
+    width: 100px;
```

I'd prefer to keep "default rules" at the beginning and eventually add media-queries mixin at the very end of the selector.

```suggestion
     width: 100px;
    
    @include media-breakpoint-up(md) {
        width: 160px;
    }
```